extensions/amp-document-recommendations/0.1/amp-document-recommendations.js

```diff
@@ -220,15 +230,42 @@ export class AmpDocumentRecommendations extends AMP.BaseElement {
       this.appendDivision_();
       this.appendArticleLinks_(this.nextArticle_ + 1);
     });
+
+    this.viewport_.onScroll(() => this.scrollHandler_());
+    this.viewport_.onResize(() => this.scrollHandler_());
   }
 
   /** @override */
   layoutCallback() {
```

Comment:
nit: this is the same as the `BaseElement` implementation, so it can be removed.

Reply:
Removed.